### PR TITLE
Sponsor page mobile style

### PR DESCRIPTION
**Pull request by lingin1230**
## Types of changes

* **New feature**

## Description
adjust sponsor page's mobile display

## Expected behavior

update  description's text content, adjust icon's size and font's line-height, size

## Related Issue

#210

### CODE REVIEW

File: i18n/sponsor/index.i18n.js [hunk 1]
```diff
@@ -37,9 +37,9 @@ export default genI18nMessages({
     'zh-hant': {
         title: '贊助召集令',
         intro: [
-            '您的支持是推進 PyCon Taiwan 的強大動力。{br}' +
-                'PyCon Taiwan 在此召集您的加入，透過贊助以具體行動支持 Python 社群的成長！',
-            '透過參與 PyCon APAC 2022，我們也提供贊助廠商以下的服務：',
+            '您的支持是推進 PyCon Taiwan 的強大動力！{br}' +
```
Comment (mattwang44) [inline i18n/sponsor/index.i18n.js:40]: Please revert the change here. The content is finalized based on [GitHub discussion](https://github.com/pycontw/pycontw-frontend/issues/206#issuecomment-1081608096).

File: pages/sponsor/index.vue [hunk 2]
```diff
@@ -7,7 +7,8 @@
                 :key="`sponsor_intro_${i}`"
                 :path="`intro.${i}`"
                 tag="p"
-                class="text-base"
+                class="text-xs md:text-sm"
+                style="line-height: 176%"
```
Comment (mattwang44) [inline pages/sponsor/index.vue:11]: Based on the current convention, please avoid inline style change. Alternatively, you can have a dedicated class and configure the css for the class in the `style` block.

**Review by mattwang44 — CHANGES_REQUESTED** [review]

File: pages/sponsor/index.vue (hunk 2 above)
Comment (lingin1230) [reply pages/sponsor/index.vue]: ok

File: pages/sponsor/index.vue [hunk 3]
```diff
@@ -115,6 +115,10 @@ export default {
 
 <style lang="postcss" scoped>
 .icon {
-    @apply w-16 md:w-24 h-16 md:h-24 bg-contain bg-no-repeat;
+    @apply w-24 md:w-24 h-24 md:h-24 bg-contain bg-no-repeat;
```
Comment (mattwang44) [inline pages/sponsor/index.vue:118]: Let's change the icon's width/height to meet the [design spec](https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=1670%3A17624).

```suggestion
    @apply bg-contain bg-no-repeat;
    width: 112px;
    height: 112px;
    @media (min-width: 768px) {
        width: 138px;
        height: 138px;
    }
```

File: pages/sponsor/index.vue [hunk 4]
```diff
@@ -115,6 +115,10 @@ export default {
...
+}
+.sponsor_context {
+    @apply text-xs md:text-sm;
+    line-height: 176%;
```
Comment (mattwang44) [inline pages/sponsor/index.vue:122]:
```suggestion
.sponsor_context {
    font-size: 16px;
    line-height: 176%;
    @media (min-width: 768px) {
        font-size: 20px;
    }
```

**Review by mattwang44 — CHANGES_REQUESTED** [review]

File: components/core/i18n/PageWrapper.vue [hunk 5]
```diff
@@ -41,7 +41,7 @@ h3 {
 
 p,
 .paragraph {
-    @apply w-full mt-1 mb-4 text-justify;
```
[the remark this answers is not in the document]
Comment (lingin1230) [reply components/core/i18n/PageWrapper.vue:44]: text-justify
text-justify 讓文字間距變得不統一，我參照 Figma 上的文字樣式是 text-left 所以先將它拿掉了，請問這邊我先將它改回來嗎？

[the remark this answers is not in the document]
Comment (mattwang44) [reply components/core/i18n/PageWrapper.vue:44]: designer 決定維持左邊~

**Review by mattwang44 — APPROVED** [review]
I've reverted the `text-justify` issue in [10ff54d](https://github.com/pycontw/pycontw-frontend/pull/211/commits/10ff54d38aeeb6359579f114d2b6d6810602813e). The rest LGTM!
